README.md

```diff
@@ -26,8 +26,47 @@ the right to make this assignment.
 
 -----
 
-Hasjob runs on [Python][] with the [Flask][] microframework. You will need
-to install all the requirements listed in `requirements.txt` using
+### Configuration
+
+Copy `settings-sample.py` to `settings.py`.
```

Comment:
docker's is called `development.docker.py`. You'll need to rename it to just `development.py`

Making a new `init_for('docker')` feels like overkill, since the goal is to make docker the standard `dev`.